samples/test/sample.test.js

```diff
@@ -186,6 +186,18 @@ describe('quickstart sample tests', () => {
     assert.include(stdout, 'relatedAsset');
   });
 
+  it('should get effective iam policies successfully', async () => {
+    const assetName = `//storage.googleapis.com/${bucketName}`;
+    let waitMs = 1000;
+    let included = false;
+    for (let retry = 0; retry < 3 && !included; ++retry) {
+      await sleep((waitMs *= 2));
+      const stdout = execSync(`node getBatchEffectiveIamPolicies ${assetName}`);
+      included = stdout.includes(assetName);
+    }
```

Comment:
Why would this need to retry? The `execSync` should fail or contain the desired output in one call

Reply:
Done